arquillian-chameleon-deployments/arquillian-chameleon-maven-build-deployment/src/main/java/org/arquillian/container/chameleon/deployment/maven/MavenBuild.java

```diff
@@ -0,0 +1,21 @@
+package org.arquillian.container.chameleon.deployment.maven;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+public @interface MavenBuild {
+
+    String[] mavenGoals() default "package";
+    String mavenVersion() default "3.5.0";
+    String[] mavenProfiles() default {};
+    String pom() default "pom.xml";
+    String localRepositoryDirectory() default "";
+    boolean offline() default false;
+    String mvnOpts() default "";
+    String[] properties() default {};
```

Comment:
I am just wondering if we really want to have `maven` prefix here. Wouldn't it be enough without it?

Reply:
in `mvnOpts` you mean?

Reply:
No, this is `mvn` prefix :) I meant ` mavenGoals`, `mavenVersion` and `mavenProfiles`.